exercises/pythagorean-triplet/src/test/java/PythagoreanTripletTest.java

```diff
@@ -20,6 +20,156 @@ public void shouldCalculateSum() {
         assertEquals(expected, actual);
     }
 
+    @Ignore("Remove to run test")
+    @Test
+    public void tripletsWhoseSumIs12() {
+        List<PythagoreanTriplet> actual
+                = PythagoreanTriplet
+                        .makeTripletsList()
+                        .withFactorsLessThanOrEqualTo(12)
+                        .thatSumTo(12)
+                        .build();
+
+        List<PythagoreanTriplet> expected
+                = Arrays.asList(
+                        new PythagoreanTriplet(3, 4, 5));
+
+        assertEquals(expected.size(), actual.size());
+        for (int i = 0; i < expected.size(); ++i) {
+            assertEquals(expected.get(i), actual.get(i));
```

Comment:
Is there a reason why you can't just compare the list here instead of iterating over it and comparing individual elements? 🙂